src/components/RepoTile/RepoTile.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import Avatar from '@components/Avatar';
+import './RepoTile.css';
+import StarIcon from '@components/StarIcon';
+import { RepoItem } from '@store/GitHubStore/types';
+
+export type RepoTileProps = {
+  item: RepoItem;
+  onClick?: (e: React.MouseEvent) => void;
+};
+
+const RepoTile: React.FC<RepoTileProps> = ({ item, onClick }) => {
+  return (
+    <div className="git-repo-tile git-repo-tile_hover" onClick={onClick}>
+      <Avatar
+        src={item.owner.avatar_url}
+        alt="avatar"
+        letter={item.owner.login[0].toUpperCase()}
```

Comment:
в таких ситуациях лучше делать проверки
```suggestion
        letter={item.owner.login.length ? item.owner.login[0].toUpperCase() : undefined}
```